gps_rviz/src/gps_handler.cpp

```diff
@@ -0,0 +1,122 @@
+#include "geometry_msgs/Vector3.h"
+#include "math.h"
+#include "visualization_msgs/Marker.h"
+#include "visualization_msgs/MarkerArray.h"
+#include "gps_handler.h"
```

Comment:
We use this include style: https://github.com/ethz-asl/multiagent_mapping/wiki/Header-include-guide
Alphabetize includes.